lib/pensio_api/request.rb

```diff
@@ -12,7 +12,7 @@ def initialize(path, options={})
     end
 
     def response_contains?(key)
-      @body && @body.has_key?(key)
+      !!@body && @body.has_key?(key)
```

Comment:
This originally returned nil instead of false - the double bang ensures its treated as a boolean